sklearn/linear_model/logistic.py

```diff
@@ -796,7 +796,7 @@ def _log_reg_scoring_path(X, y, train, test, pos_class=None, Cs=10,
         values are chosen in a logarithmic scale between 1e-4 and 1e4.
         If not provided, then a fixed set of values for Cs are used.
 
-    scoring : callable
+    scoring : callable or a string present in SCORERS
```

Comment:
use the same docstring as in:

https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/model_selection/_validation.py#L87